server/config.py

```diff
@@ -1,79 +1,31 @@
-logging.addLevelName(TRACE, 'TRACE')
-logging.getLogger('aiomeasures').setLevel(logging.INFO)
+logging.addLevelName(TRACE, "TRACE")
+logging.getLogger("aiomeasures").setLevel(logging.INFO)
 
-# Environment
-LOG_LEVEL = logging.getLevelName(os.getenv('LOG_LEVEL', 'DEBUG'))
-PROFILING_INTERVAL = int(os.getenv('PROFILING_INTERVAL', -1))
-
-# Credit to Axle for parameter changes, see: http://forums.faforever.com/viewtopic.php?f=45&t=11698#p119599
+# Credit to Axle for parameter changes,
+# see: http://forums.faforever.com/viewtopic.php?f=45&t=11698#p119599
 # Optimum values for ladder here, using them for global as well.
 trueskill.setup(mu=1500, sigma=500, beta=240, tau=10, draw_probability=0.10)
 
-METRICS_PORT = int(os.getenv('METRICS_PORT', 8011))
-ENABLE_METRICS = os.getenv('ENABLE_STATSD', 'false').lower() == 'true'
-
-WWW_URL = 'https://www.faforever.com'
-CONTENT_URL = 'http://content.faforever.com'
-
-DB_SERVER = os.getenv("DB_PORT_3306_TCP_ADDR", "127.0.0.1")
-DB_PORT = int(os.getenv("DB_PORT_3306_TCP_PORT", "3306"))
-DB_LOGIN = os.getenv("FAF_DB_LOGIN", "root")
-DB_PASSWORD = os.getenv("FAF_DB_PASSWORD", "banana")
-DB_NAME = os.getenv("FAF_DB_NAME", "faf")
-
-API_CLIENT_ID = os.getenv("API_CLIENT_ID", "client_id")
-API_CLIENT_SECRET = os.getenv("API_CLIENT_SECRET", "banana")
-API_TOKEN_URI = os.getenv("API_TOKEN_URI", "https://api.test.faforever.com/oauth/token")
-API_BASE_URL = os.getenv("API_BASE_URL", "https://api.test.faforever.com/")
-USE_API = os.getenv("USE_API", 'true').lower() == 'true'
-
-FAF_POLICY_SERVER_BASE_URL = os.getenv("FAF_POLICY_SERVER_BASE_URL", "http://faf-policy-server")
-FORCE_STEAM_LINK_AFTER_DATE = int(os.getenv('FORCE_STEAM_LINK_AFTER_DATE', 1536105599))  # 5 september 2018 by default
-FORCE_STEAM_LINK = os.getenv('FORCE_STEAM_LINK', 'false').lower() == 'true'
-
-NEWBIE_BASE_MEAN = int(os.getenv('NEWBIE_BASE_MEAN', 500))
-NEWBIE_MIN_GAMES = int(os.getenv('NEWBIE_MIN_GAMES', 10))
-TOP_PLAYER_MIN_RATING = int(os.getenv('TOP_PLAYER_MIN_RATING', 1600))
-
-TWILIO_ACCOUNT_SID = os.getenv("TWILIO_ACCOUNT_SID", "")
-TWILIO_TOKEN = os.getenv("TWILIO_TOKEN", "")
-TWILIO_TTL = os.getenv("TWILIO_TTL", 3600*24)
-
-COTURN_HOSTS = list(filter(None, os.getenv('COTURN_HOSTS', "").split(',')))
-COTURN_KEYS = list(filter(None, os.getenv('COTURN_KEYS', "").split(',')))
 
-GEO_IP_DATABASE_PATH = os.getenv("GEO_IP_DATABASE_PATH", "GeoLite2-Country.mmdb")
-GEO_IP_DATABASE_URL = os.getenv(
-    "GEO_IP_DATABASE_URL",
-    "https://download.maxmind.com/app/geoip_download"
-)
-GEO_IP_LICENSE_KEY = os.getenv("GEO_IP_LICENSE_KEY")
-GEO_IP_DATABASE_MAX_AGE_DAYS = int(os.getenv('GEO_IP_DATABASE_MAX_AGE_DAYS', 22))
+class ConfigurationStore:
+    def __init__(self):
+        self.refresh()
 
-CONTROL_SERVER_PORT = int(os.getenv('CONTROL_SERVER_PORT', 4000))
+    def refresh(self):
+        config_file = os.getenv("CONFIGURATION_FILE", "conf.yaml")
+        with open(config_file, "r") as f:
+            config_dict = yaml.safe_load(f)
```

Comment:
Maybe we rename the conf file that's checked in to `default_conf.yaml` and do something like this:
```suggestion
        with open("default_conf.yaml") as f:
            config_dict = yaml.safe_load(f)
        config_file = os.getenv("CONFIGURATION_FILE")
        if config_file:
            with open(config_file) as f:
                config_dict.update(yaml.safe_load(f))
```

That way we'll always have a default value checked in to the repo.